mhs/mhs/common/state/dynamo_persistence_adaptor.py

```diff
@@ -0,0 +1,112 @@
+import json
+import traceback
+
+import aioboto3
+import utilities.integration_adaptors_logger as log
+
+import mhs.common.state.persistence_adaptor
+
+logger = log.IntegrationAdaptorsLogger('DYNAMO_PERSISTENCE')
+
+
+class RecordCreationError(RuntimeError):
+    def __init__(self):
+        pass
+
+
+class RecordDeletionError(RuntimeError):
+    def __init__(self):
+        pass
+
+
+class RecordRetrievalError(RuntimeError):
+    def __init__(self):
+        pass
+
+
+class InvalidTableError(RuntimeError):
+    def __init__(self):
+        pass
+
+
+class DynamoPersistenceAdaptor(mhs.common.state.persistence_adaptor.PersistenceAdaptor):
```

Comment:
Yes, and no.

I have used very similar code previously. But I have not run this specific version of it. I will do so before actually merging this branch into 'develop' so that any required changes can be made.

Reply:
Yes,